### ISSUE TITLE
Getting an error inside the area function

### ISSUE THREAD

**Issue by agostini01** [aditipurandare/circuitRouting#3]
Command:
```
python parsing/parsingCode.py -i samples/parsingSample.txt
```

Output
```
['', 'R1-2', 'BT1-1', 'D1-1', 'BT1-2', 'R1-1', 'D1-2']
['R1-2', 0, 1, 0, 0, 0, 0]
['BT1-1', 1, 0, 0, 0, 0, 0]
['D1-1', 0, 0, 0, 1, 0, 0]
['BT1-2', 0, 0, 1, 0, 0, 0]
['R1-1', 0, 0, 0, 0, 0, 1]
['D1-2', 0, 0, 0, 0, 1, 0]
[['R1 ', '134580 ', '-98280 '], ['D1 ', '146340 ', '-92920 '], ['BT1 ', '137160 ', '-95250 ']]
Traceback (most recent call last):
  File "parsing/parsingCode.py", line 188, in <module>
    main()
  File "parsing/parsingCode.py", line 184, in main
    area=get_area(args.filename)
  File "parsing/parsingCode.py", line 115, in get_area
    for x in range(0,len(lines)-1):
NameError: name 'lines' is not defined
```

**Comment by agostini01** [comment]
Now I am getting another error.

```
python parsing/parsingCode.py -i samples/parsingSample.txt
```

```
['', 'R1-2', 'BT1-1', 'D1-1', 'BT1-2', 'R1-1', 'D1-2']
['R1-2', 0, 1, 0, 0, 0, 0]
['BT1-1', 1, 0, 0, 0, 0, 0]
['D1-1', 0, 0, 0, 1, 0, 0]
['BT1-2', 0, 0, 1, 0, 0, 0]
['R1-1', 0, 0, 0, 0, 0, 1]
['D1-2', 0, 0, 0, 0, 1, 0]
[['R1 ', '134580 ', '-98280 '], ['D1 ', '146340 ', '-92920 '], ['BT1 ', '137160 ', '-95250 ']]
Traceback (most recent call last):
  File "parsing/parsingCode.py", line 194, in <module>
    main()
  File "parsing/parsingCode.py", line 190, in main
    area=get_area(args.filename)
  File "parsing/parsingCode.py", line 129, in get_area
    xVals[n].append(lines[x][:l1])
IndexError: list index out of range
```

Also, once you feel that you wrote code that resolves an issue on github, you can write in your commit message:

```
Fix <gituser>/<gitrepo>#<issuenumber>

as in:

Fix  aditipurandare/circuitRouting#3
```

If you just want to update but not fix, you can write:

```
Update aditipurandare/circuitRouting#3
```

**Comment by agostini01** [comment]
I made the code print the line it is parsing 

```
            l1 = lines[x].find(" ")
            print(lines[x])
            print(l1)
            xVals[n].append(lines[x][:l1])
```

And the code breaks at this point on the file of parsing:

```
  (library

    (image Resistor_THT:R_Axial_DIN0204_L3.6mm_D1.6mm_P1.90mm_Vertical

      (outline (path signal 50  2860 1050  -1050 1050))

```

With the following printed statements for lines[x] and l1

```
2860 1050  -1050 1050))

4
```

Maybe there is an error on unix vs linux?

Can you try to debug the actual .py file provided in this project instead of the notebook?
